pkgs/development/beam-modules/rebar3-release.nix

```diff
@@ -6,6 +6,7 @@
 }:
 
 { name
```

Comment:
I think we should make `name` optional

Reply:
No idea what the idiomatic way of doing this is though :D

Reply:
```nix
name = "${name}-${version}"
```

Reply:
I agree with making name optional. The only problem is what happens in the meanwhile, while there are still packages that have a name and no pname. Should we change those at the same time? (like erlang-ls...)
I made name default to null. Perhaps we can remove it as an input parameter altogether and change the other packages?
It would be a breaking change for everybody else using this.

Reply:
> Should we change those at the same time?

That would be the easiest and clean it up. We can keep name and convert it into a throw if you want.

Reply:
Trying to remove name, let's see what the ci says.
@dlesl since you are probably going to be one affected by removing name, will you be ok? I don't feel it's a big change, but just want to be sure.

Reply:
Yea I think it's fine, it should be a simple fix for anyone affected